<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCardWindow</class>
 <widget class="QDialog" name="AddCardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new card</string>
  </property>
  <widget class="QLineEdit" name="lineForPhrase1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>Enter Text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineForPhrase2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>Enter Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="Phrase1Label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Phrase1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Phrase2Label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Phrase2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="HardRB">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="MediumRB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="EasyRB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QLabel" name="PriorityLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Difficulty</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>269</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HintLabel1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>45</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Allowed symbols: letters and  ',.-?!' ' Letter case matters. 
Max size - 20 symbols. First symbol can be only a letter.</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HintLabel2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>155</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Allowed symbols: letters and  ',.-?!' ' Letter case matters. 
Max size - 20 symbols. First symbol can be only a letter.</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
